nxc/modules/powershell_history.py

```diff
@@ -0,0 +1,84 @@
+import traceback
+import os
+from impacket.examples.secretsdump import RemoteOperations
+
+class NXCModule:
+    """Module by @357384n"""
+
+    name = "powershell_history"
+    description = "Extracts PowerShell history for all users and looks for sensitive commands."
+    supported_protocols = ["smb"]
+    opsec_safe = True
+    multiple_hosts = True
+
+    def options(self, context, module_options):
+        """To export all the history you can add the following option: -o export=enable"""
```

Comment:
Probably `export=True` would be better, than simply `if self.export: ...` could be used later